### PR TITLE
docs: added more detailed docs for toleration

### PR DESCRIPTION
**Pull request by h4ck3rk3y**
Docs weren't up to standards! Improved it

### CODE REVIEW

File: docs/docs/api-reference/starlark-reference/toleration.md [hunk 1]
```diff
@@ -8,10 +8,32 @@ can be used with a [ServiceConfig][service-config] object.
...
+    # OPTIONAL
     key = "key",
+
+    # operator represents a key's relationship to the value.
+    # Valid operators are Exists and Equal. Defaults to Equal.
+    # Exists is equivalent to wildcard for value, so that a pod can tolerate all taints of a particular category.
+    # OPTIONAL
     operator = "Equal",
+
+	# value is the taint value the toleration matches to.
```
Comment (leoporoli) [inline docs/docs/api-reference/starlark-reference/toleration.md:22]: Extra indentation here and in the following fields' comments

**Review by leoporoli — APPROVED** [review]

File: docs/docs/api-reference/starlark-reference/toleration.md (hunk 1 above)
Comment (h4ck3rk3y) [reply docs/docs/api-reference/starlark-reference/toleration.md:22]: put tabs now! does it look better?

File: docs/docs/api-reference/starlark-reference/toleration.md [hunk 2]
```diff
@@ -8,11 +8,33 @@ can be used with a [ServiceConfig][service-config] object.
 
 ```python
 toleration = Toleration(
```
Comment (barnabasbusa) [inline docs/docs/api-reference/starlark-reference/toleration.md:10]:
```suggestion
toleration = [
Toleration(..)
]
```

Comment (h4ck3rk3y) [reply docs/docs/api-reference/starlark-reference/toleration.md:10]: Hey! This is an individual toleration. It is in a list in ServiceConfig!

**Pull request merged**